lib/ruumba/analyzer.rb

```diff
@@ -107,12 +109,49 @@ def copy_erb_files(fq_dir, tmp, pwd)
     end
 
     def run_rubocop(target, tmp)
-      args = (@options[:arguments] || []).join(' ')
+      args = ['rubocop'] + (@options[:arguments] || []) + [target.to_s]
       todo = tmp + '.rubocop_todo.yml'
 
-      system("cd #{tmp} && rubocop #{args} #{target}").tap do
-        FileUtils.cp(todo, ENV['PWD']) if File.exist?(todo)
-        FileUtils.rm_rf(tmp) unless @options[:tmp_folder]
+      pwd = ENV['PWD']
+
+      replacements = []
+
+      if args.include?('--format')
+        replacements << [/^#{Regexp.quote(tmp.to_s)}/, pwd]
+      end
+
+      unless @options[:disable_rb_extension]
+        replacements << [/\.erb\.rb/, '.erb']
+      end
+
+      Dir.chdir(tmp) do
```

Comment:
Should we bubble up the rubocop return code and make that the return code from ruumba?

Reply:
I've added this now.